recipe/meta.yaml

```diff
@@ -23,14 +21,16 @@ requirements:
 
   host:
     - python
+    - pip
     - setuptools
-    - numpy 1.11.*
+    - numpy
+    - numpy >=1.8.0
```

Comment:
@mbargull thank you for help. Is there any special reason that numpy is listed twice under host or can I remove the first line?

Reply:
The first line, i.e., `- numpy`, is the important one that will be replaced with `- numpy 1.14` via https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/f6e8a2fb2eb19a98b26474efc1fb89a6bf6bcc17/recipe/conda_build_config.yaml#L530. Don't remove it ;).

The second one, i.e., `- numpy >=1.8.0`, is technically not needed since we always have never `numpy` versions anyway. I just like to add it for documentation purposes since it's the requirement specified upstream in `setup.py`.

Reply:
Ok, thank you for explaining. :)